src/test_l1_dsHdmiIn.c

```diff
@@ -1730,20 +1727,27 @@ void test_l1_dsHdmiIn_negative_dsGetHDMISPDInfo(void) {
 void test_l1_dsHdmiIn_positive_dsSetEdidVersion(void) {
     gTestID = 37;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    uint8_t ver14, ver20;
+
+    ver14 = ut_kvp_getUInt8Field(ut_kvp_profile_getInstance(),"dsHdmiIn/EdidVersions/0");
```

Comment:
There are macros to use` ut_kvp_profile_getInstance()`

`https://github.com/rdkcentral/ut-core/blob/develop/include/ut_kvp_profile.h`

so

`ut_kvp_getUInt8Field(ut_kvp_profile_getInstance(),`

can become

`UT_ASSERT_KVP_EQUAL_PROFILE_UINT8(`

Reply:
done